Packages/Sources/AlchemyAPIClient/AlchemyAPIClient.swift

```diff
@@ -0,0 +1,40 @@
+import DataModel
+import Dependencies
+import DependenciesMacros
+
+@DependencyClient
+public struct AlchemyAPIClient: Sendable {
+    private var apiKey = APIKey("<-- undefined -->")
+    private var user = ETHAddress("<-- undefined -->")
+    
+    public internal(set) var onGetNFTsForOwner: @Sendable (APIKey, ETHAddress, PageKey?) async throws -> OwnerNFTPage
+    
```

Comment:
I generally like to keep dependency endpoint access hidden outside the package (just internal(set)) and add a public method that calls the closure, but @DependencyClient needs it to be public to generate public initializer